### PR TITLE
remove old k/federation jobs

### PR DESCRIPTION
**Pull request by krzyzacy**
looks like https://github.com/kubernetes/federation/commits/master doesn't have new commits for the past 3 months? is the v1 federation project still alive? @shashidharatd 

/area jobs
/assign @BenTheElder

### CODE REVIEW

**Comment by shashidharatd** [comment]
yes @krzyzacy, federation v1 is not maintained and we have active development in federation v2 https://github.com/kubernetes-sigs/federation-v2

/cc @kubernetes/federation-maintainers

**Comment by krzyzacy** [comment]
@shashidharatd sounds like I'll just delete those jobs?

File: prow/cmd/tot/main_test.go [hunk 1]
```diff
@@ -193,11 +193,11 @@ func TestGetURL(t *testing.T) {
 		{
 			name: "fallback bucket - postsubmit",
 
-			jobName: "ci-federation-release",
+			jobName: "ci-test-infra-bazel",
```
Comment (BenTheElder) [inline prow/cmd/tot/main_test.go]: oh gross

Comment (krzyzacy) [reply prow/cmd/tot/main_test.go]: /shrug

**Pull request merged**